surfstudio/ios-kinescope-sdk
# Add plist reader

## Changes

- Add `ConfigStorage` wrapper for read `KinescopeConfig.plist` file from disk and parse into `User` model.
- Add readme

## How to test?

- Add `KinescopeConfig.plist` (see readme) ->  Run application -> set breakpoint in `setConfig` SDK's method -> check the `apiKey`
